src/interfaces/assistants_web/src/utils/citations.ts

```diff
@@ -5,7 +5,7 @@ import { Citation } from '@/cohere-client';
  * This is a temporary fix as the current model is consistently returning incorrect markdown image syntax.
  * @param text - message text or citation text
  */
-export const fixMarkdownImagesInText = (text: string) => {
```

Comment:
The name is descriptive, since I am planning to create another function to fix the markdown elments